core/sis-referencing/src/test/java/org/apache/sis/geometry/ArrayEnvelopeTest.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.sis.geometry;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Unit tests for class {@link ArrayEnvelope}.
+ *
+ * @version 0.8
+ * @since   0.8
+ * @author  Michael Hauseggeer, <hausegger.michael@googlemail.com>
+ * @see ArrayEnvelope
+ *
+ */
+public class ArrayEnvelopeTest{
+  @Test
+  public void testIsEmptyReturningTrue() {
+      double[] doubleArray = new double[2];
```

Comment:
A variable name that just repeat the type name is not very useful. It is more useful to use a variable name telling what the data are. In this case, `coordinate`.